FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,40 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let this = self else { return }
+			switch result {
+			case .success((let data, let response)):
+				if response.statusCode == this.ok_200, let feedImages = try? JSONDecoder().decode(FeedImageParser.self, from: data).feedImages {
+					completion(.success(feedImages))
+				} else {
+					completion(.failure(RemoteFeedLoader.Error.invalidData))
+				}
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
Since this portion of code is already inside the `RemoteFeedLoader` scope, we can make the error shorter:
```suggestion
				completion(.failure(Error.connectivity))
```